lld/test/wasm/lto/stub-library.s

```diff
@@ -16,6 +16,23 @@
 # RUN: wasm-ld %t.o %t/libfoo.a %p/Inputs/stub.so -o %t2.wasm
 # RUN: obj2yaml %t2.wasm | FileCheck %s
 
+## The function `foo` is declared in stub2.so and depends on `baz`, and both
+## `foo` and `baz` are defined in an LTO object. When `foo` and `baz` are
+## DCE'd and become undefined in the LTO process, wasm-ld should not try to
+## export the (nonexistent) `baz`.
+
+# RUN: llvm-mc -filetype=obj -triple=wasm32-unknown-unknown -o %t.o %s
+# RUN: mkdir -p %t
+# RUN: llvm-as %S/Inputs/foo.ll -o %t/foo.o
```

Comment:
Should this bee foobaz.ll here and above?

Reply:
Fixed: https://github.com/llvm/llvm-project/pull/173422/commits/9545345cc957eedbdedc8c1d675cf8d8af01e0ab